opennms-base-assembly/src/main/filtered/etc/events/opennms.internal.events.xml

```diff
@@ -248,10 +248,25 @@
    <event>
       <uei>uei.opennms.org/internal/applicationDeleted</uei>
       <event-label>OpenNMS-defined application event: applicationDeleted</event-label>
-      <descr>&lt;p>Application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been deleted.&lt;/p></descr>
-      <logmsg dest="logndisplay">Application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been deleted.</logmsg>
+      <descr>Application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been deleted.</descr>
+      <logmsg dest="logndisplay">Application '%parm[applicationName]%' has been deleted.</logmsg>
       <severity>Warning</severity>
    </event>
+   <event>
+      <uei>uei.opennms.org/internal/applicationChanged</uei>
+      <event-label>OpenNMS-defined node event: applicationChanged</event-label>
+      <descr>The application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been changed.</descr>
+      <logmsg dest="logndisplay">Application '%parm[applicationName]%' configuration has been changed.</logmsg>
+      <severity>Warning</severity>
+      <operinstruct>Make sure the application '%parm[applicationName]%' still works properly. Please verify it's &lt;a href=&quot;admin/applications.htm?applicationid=%parm[applicationId]%&amp;edit=services&quot;>definition&lt;/a>.</operinstruct>
```

Comment:
What do you mean by making sure the application works properly?

Reply:
Maybe a bit confusing, right. People should get a hint to check the configuration after a change. Maybe now it does not monitor anymore as expected or initially designed.